scripts/setup_roboschool.sh

```diff
@@ -0,0 +1,37 @@
+echo "Make sure this script is running under garage conda environment"
+
+# install dependency
+if [[ "$(uname)" == "Darwin" ]]; then
+	brew install cmake tinyxml assimp ffmpeg
+	brew reinstall boost-python3 --without-python --with-python3 --build-from-source
+	conda install qt
```

Comment:
just add this to environment.yml if it is required

Reply:
Preferably use a pip dependency instead of a conda dependency. Pin it to at least a major version (e.g. Qt 3, 4, and 5 are very different).